posthog/hogql/grammar/HogQLParser.g4

```diff
@@ -0,0 +1,228 @@
+parser grammar HogQLParser;
+
+options {
+    tokenVocab = HogQLLexer;
+}
+
+// SELECT statement
```

Comment:
The select statements are not supported yet, and any query that tries to use them will thrown an error (see `parser.py` later). I'm keeping them here regardless, and will keep filling this skeleton in the next PRs.